packages/backend/src/managers/playground.ts

```diff
@@ -213,17 +213,22 @@ export class PlayGroundManager {
           if (!q) {
             throw new Error('query not found in state');
           }
+          q.error = undefined;
           q.response = result as ModelResponse;
           this.queries.set(query.id, q);
-          this.sendQueriesState().catch((err: unknown) => {
-            console.error('playground: unable to send the response to the frontend', err);
-          });
+          void this.sendQueriesState();
         }
       });
     });
     // post the data
     post_req.write(post_data);
     post_req.end();
+    post_req.on('error', error => {
+      console.error('connection on error.');
```

Comment:
Maybe you can add the error here

Reply:
Yes, I forgot apparently, thanks for noticing !